Gemfile

```diff
@@ -85,6 +85,7 @@ gem 'zendesk_api'
 gem 'viewpoint'
 gem 'rubyntlm', git: 'https://github.com/wimm/rubyntlm.git'
 gem 'autodiscover', git: 'https://github.com/thorsteneckel/autodiscover.git'
+gem 'wysiwyg-rails'
```

Comment:
Since Zammad is a web app, we try to do all dependencies for the web app not in rails context (we add no gems for adding java script libs). The reason is, in one of our next steps we want to separate the REST store from the web app.